larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -508,6 +508,19 @@ private int reportFailedCompare(TestExecutionObserver testExecutionObserver, Sce
 
 	public String prepareResultForCompare(String input, Properties properties, Map<String, Map<String, Map<String, String>>> ignoreMap) {
 		String result = input;
+
+		// TESTDATA-dir based file paths in results can often not be compared properly because of location differences, and complicated by system-dependent file paths.
+		// Changing Windows backslashes to Unix forward slashes in the rest of file paths is done with regexes configured in global.properties
+		String testDataDirPropertyValue = AppConstants.getInstance().getProperty("testdata.dir");
```

Comment:
Dit is wel een iaf-test specifiek piefje hoor... Misschien dat elke directory standaard genormalizeerd moet worden?

Reply:
Maar hoe vind je die directories / paden in de output? Op een gegarandeerde manier?
De regex die we hadden, die werkt in ieder geval niet goed omdat er teveel variatie is in hoe paden gespecifeerd kunnen worden, en teveel verschillende plaatsen waar een pad ergens in de config komt, uit de config wordt gehaald, of in een Message kan komen.

Ik zie voorlopig nog geen goede manier om paden altijd gestandaardiseerd te krijgen.

Reply:
alles wat een absolute path (rare property methode) krijgt normaliseren?